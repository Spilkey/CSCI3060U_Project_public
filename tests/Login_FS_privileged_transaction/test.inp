login
user02
create
user01
admin
logout