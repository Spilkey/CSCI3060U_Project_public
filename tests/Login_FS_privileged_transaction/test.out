01_user01__________AA_000000000
00_user02__________FS_000000000